lib/util/url.ts

```diff
@@ -1,6 +1,9 @@
 import urlJoin from 'url-join';
 import { regEx } from './regex';
 
+const ensureTrailingSlashRe = /\/?$/;
+const trimTrailingSlashRe = regEx(/\/+$/);
+const trimLeadingSlashRe = regEx(/^\//);
 export function joinUrlParts(...parts: string[]): string {
```

Comment:
```suggestion

export function joinUrlParts(...parts: string[]): string {
```

Wrongly reverted? 🙃

Reply:
It broke the code last time so I was trying to check if just reverting it works or not.

I have found a trail that . The files in  `lib/manager/terraform` have coverage issues and am looking in to find the exact problem.

Reply:
I don't think a newline brokes it. 😉